packages/cli/src/utils/compiler.ts

```diff
@@ -95,16 +97,26 @@ export const compilePlatformApp = async ({ outputName, entryFile, projectPath =
         build: {
             lib: {
                 entry: entryFile,
-                name: 'Settings',
-                formats: ['es'],
+                name: outputName,
+                formats: ['iife'],
```

Comment:
We are looking to move to `es` when in production mode for the blocks and themes, why did you have to change for the platform apps?

Reply:
we re-implemented how themes are calling the settings. By sticking it on the window and then directly executing allows us to access window.default.

We tried to export it as lib -> format: 'es' and using the import() to dynamically import the settings file didnt work well for us. We had cors issues and couldnt make it work..